src/modules/logicModule/LM_PC_RecurringPayments_v1.sol

```diff
@@ -96,6 +96,9 @@ contract LM_PC_RecurringPayments_v1 is
 
     /// @dev	Marks the beginning of the list.
     uint internal constant _SENTINEL = type(uint).max;
+    /// @dev	Flag config for payment processor indicates presence of start, end and cliff.
+    bytes16 internal constant _START_END_CLIFF_FLAG =
```

Comment:
This is of course a quicker way than going with the ">>"'s , but I think we should be consistent across our clients if we go with one or another. Which one do you think is best?

Reply:
Per se I think it would be fine to use both, if the constant values are explained well (better than they are right now), but now that I look at this specific instance, I think it might make more sense to have the flags assembled "manually" in the init so that if we want to make the recurring payment processor more flexible in the future (enabling it to use different flag configs) we don't have to touch the contract state. 

So I'm setting the flags manually now.